sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStoreUtils.java

```diff
@@ -256,6 +257,36 @@ private static void setBooleanProperty(StateStore stateStore, String propertyNam
     }
 
     /**
+     * Stores a TaskInfo as a Property in the state store.
+     * @param stateStore StateStore to store in.
+     * @param taskName Name of the task
+     * @param taskInfo TaskInfo to store.
+     */
+    public static void storeTaskStatusAsProperty(StateStore stateStore, String taskName, TaskStatus taskInfo) {
+        stateStore.storeProperty(taskName + PROPERTY_TASK_INFO_APPENDIX, taskInfo.toByteArray());
```

Comment:
Disagree, it is a bad pattern to log exceptions at this layer (see other methods in this utils class)

For simple get/sets like this the caller should handle any exceptions. 

Probably more useful to add `throws`.